src/utils/__tests__/create-file.ts

```diff
@@ -4,7 +4,7 @@ import { createFile } from "../create-file";
 
 const mkdir = promisify(fs.mkdir);
 const rm = promisify(fs.rm);
-const tmpDir = "./.tmp";
+const tmpDir = "./.tmp-create-file";
```

Comment:
Two tests used the same `./.tmp`. This created a bit of test instability when they were both trying to create/delete the folder at the same time. Using different names solves it.

Reply:
Thanks! I had seem problems from this, never figured out the cause.